DQMOffline/Trigger/plugins/HTDQM.cc

```diff
@@ -0,0 +1,83 @@
+#include "DQMOffline/Trigger/plugins/HTDQM.h"
+
+HTDQM::HTDQM(){}
+
+HTDQM::~HTDQM(){}
+
+void HTDQM::initialise(const edm::ParameterSet& iConfig ){
+
+  ht_variable_binning_ = iConfig.getParameter<edm::ParameterSet>("histoPSet").getParameter<std::vector<double> >("htBinning");
+  met_variable_binning_ = iConfig.getParameter<edm::ParameterSet>("histoPSet").getParameter<std::vector<double> >("metBinning2");
+  ht_binning_ = getHistoPSet(iConfig.getParameter<edm::ParameterSet>("histoPSet").getParameter<edm::ParameterSet>("htPSet"));
+  ls_binning_ = getHistoLSPSet (iConfig.getParameter<edm::ParameterSet>("histoPSet").getParameter<edm::ParameterSet>("htlsPSet"));
+  
+  htME_variableBinning_.numerator   = nullptr;
```

Comment:
Hi @davidlange6, I can't find initialization in [DQMOffline/Trigger/plugins/TriggerDQMBase.h](https://github.com/cms-sw/cmssw/blob/master/DQMOffline/Trigger/plugins/TriggerDQMBase.h#L20-L27).
Do you want to call clear function like `htME_variableBinning_.clear;` instead of nullptr?